### PR TITLE
Create a guide to the design system monthly chat

### PR DESCRIPTION
**Pull request by kellylee-gds**

### CODE REVIEW

File: source/community/a-guide-to-the-design-system-monthly-chat.html.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## How to promote the session
+Comms and tickets for the session should be published at least two weeks in advance of the call. This is to give community members enough notice to attend.
+
+Comms are sent out via the mailing list on Mailchimp, and on our Slack channels. Tickets are created via Eventbrite.
+
+## How to prepare for the session
+Before the event, ensure all volunteer roles are filled. This is the responsibility of the delivery manager assigned to community work.
+
+We should aim to have prepared the slide deck a week before the session so it can be shared with attendees in advance. This is generally best accessibility practice.
+
```
Comment (trang-erskine) [inline source/community/a-guide-to-the-design-system-monthly-chat.html.md:68]:
```suggestion
If you are presenting or doing a demo, you should do a practice run before the session to check you are within the time allocated. You should check links and email addresses are correct and working. 
```

File: source/community/a-guide-to-the-design-system-monthly-chat.html.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+If you are due to speak at the event, it is advised that you access Zoom via the desktop app rather than the browser. This is because accessing Zoom through a browser will prevent you from being able to “drive” the slide deck during your presentation. 
+If using the desktop app, ensure the correct permissions are in place on your PC so you’re able to screen share, and share audio and video. You can do this via going to ‘System Settings’ — ‘Privacy & Security’ — ‘Screen and System Audio Recording’ and enabling permission for Zoom. You may need to restart Zoom for this to take effect. Follow the same steps for enabling ‘Microphone’ and ‘Video” within ‘Privacy & Security’.
+
+If speaking, it’s helpful to have a quiet room to present from to limit background noise. It’s also helpful to have your speaker notes in the main slide deck. During the session, the host will be able to give you co-presenter permissions which enables you to view your speaker notes on the same screen as the slide.
```
Comment (trang-erskine) [inline source/community/a-guide-to-the-design-system-monthly-chat.html.md:72]: Does this work? "During the session, the host will be able to give you co-presenter permissions which enables you to view your speaker notes on the same screen as the slide."

I couldn't work out how to get speaker notes up when I did my presentation for the first DS Day.

**Review by trang-erskine — APPROVED** [review]
Looks fine. Added an extra line and had a questions about co-presenting.

File: source/community/a-guide-to-the-design-system-monthly-chat.html.md (hunk 2 above)
Comment (kellylee-gds) [reply source/community/a-guide-to-the-design-system-monthly-chat.html.md:72]: It's definitely not obvious!

**Pull request merged**